metals/src/main/scala/scala/meta/internal/builds/BuildTools.scala

```diff
@@ -117,6 +125,7 @@ object BuildTools {
     new BuildTools(
       workspace,
       Nil,
-      () => UserConfiguration()
+      () => UserConfiguration(),
+      false
```

Comment:
Actually I'll need to change this to `true` in order for the tests to pass. In reality the way we treat Bill is a bit not like we actually treat BSP servers in that we expect to be able to auto-connect to Bill right away without any choice been made. So I'll change this to `true` to "mimic" the actual behaviour that would happen in real life if we expected the ability to auto-connect to a new `.bsp/entry`.

Reply:
Actually this was incorrect as well. Instead I just needed to add another check. So basically there are 3 things that make the BSP connection auto-connectable:

1. There is already a `.bloop/`
2. There is a `.bsp/<entry>.json` and an explicit choice was made already to use it
3. There is a `.bsp/<entry>.json` and it's a build tool we don' have support for (Like Bill).